presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -319,69 +324,69 @@ Configuration Properties
 
                                                         Example: ``/etc/hadoop/conf/core-site.xml.`` This property
                                                         is required if the iceberg.catalog.type is ``hadoop``.
                                                         Otherwise, it will be ignored.
 
-``iceberg.file-format``                                 The storage file format for Iceberg tables. The available     ``PARQUET``
+``iceberg.file-format``                                 The storage file format for Iceberg tables. The available     ``PARQUET``                        Yes                 NA
                                                         values are ``PARQUET`` and ``ORC``.
 
-``iceberg.compression-codec``                           The compression codec to use when writing files. The          ``GZIP``
+``iceberg.compression-codec``                           The compression codec to use when writing files. The          ``GZIP``                           Yes                 NA
                                                         available values are ``NONE``, ``SNAPPY``, ``GZIP``,
                                                         ``LZ4``, and ``ZSTD``.
 
-``iceberg.max-partitions-per-writer``                   The Maximum number of partitions handled per writer.          ``100``
+``iceberg.max-partitions-per-writer``                   The maximum number of partitions handled per writer.          ``100``                            Yes                 NA
 
-``iceberg.minimum-assigned-split-weight``               A decimal value in the range (0, 1] is used as a minimum      ``0.05``
+``iceberg.minimum-assigned-split-weight``               A decimal value in the range (0, 1] is used as a minimum      ``0.05``                           Yes                 Yes
                                                         for weights assigned to each split. A low value may improve
                                                         performance on tables with small files. A higher value may
                                                         improve performance for queries with highly skewed
                                                         aggregations or joins.
 
-``iceberg.enable-merge-on-read-mode``                   Enable reading base tables that use merge-on-read for         ``true``
+``iceberg.enable-merge-on-read-mode``                   Enable reading base tables that use merge-on-read for         ``true``                           Yes                 Yes, only needed on coordinator
                                                         updates.
 
-``iceberg.delete-as-join-rewrite-enabled``              When enabled, equality delete row filtering is applied        ``true``
+``iceberg.delete-as-join-rewrite-enabled``              When enabled, equality delete row filtering is applied        ``true``                           Yes                 Yes
```

Comment:
I don't think Presto C++ supports it. Presto C++ do not interpret the equality delete file as joins, but would compile it into domain filters or filter functions. Also, reading equality delete file PR is not merged yet. Just say No for now.

Reply:
Sure, I missed equality delete here.